package.json

```diff
@@ -77,7 +77,7 @@
     "striptags": "^3.1.1",
     "uuid": "^8.3.2",
     "winston": "^3.3.3",
-    "xml2js": "^0.4.23",
+    "xml2js": "^0.6.2",
```

Comment:
The vulnerability was technically fixed in 0.5.x, but it broke the `.hasOwnProperty` prototype method you're using in this codebase already. Therefore, based on the repository owner's [recommendation](https://github.com/Leonidas-from-XIV/node-xml2js/issues/672#issuecomment-1651509676), I bumped to 0.6.x which fixes the CVE but in a backward compatible way with 0.4.x.